.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,13 +10,44 @@
   - when this PR is merged.
   -->
 
+## How to test this PR?
+<!-- Mention steps about how reviewer can verify your work. -->
+
+## Applicable for UI specific PR
```

Comment:
This section should come last. I would suggest phrasing it as:

```
## For UI-specific PRs only
<!-- Delete this section if this PR does not include UI-related changes. -->

### Espresso Test Screenshots
<!-- ... -->

### Accessibility checks
<!-- Explain how this PR has been tested for accessibility, or why doing so is not applicable. For more info, check out the [Oppia Android Accessibility Guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide). -->
```

(I moved RTL and screenshots/recordings to the "proof" section.)

Reply:
Done.